certification/internal/policy/operator/scorecard_basic_check_spec.go

```diff
@@ -69,8 +68,7 @@ func (p *ScorecardBasicSpecCheck) Help() certification.HelpText {
 		}
 	}
 	return certification.HelpText{
-		Message: "Check ScorecardBasicSpecCheck encountered an error. Please review the " +
-			artifacts.Path() + "/" + scorecardBasicCheckResult + " file for more information.",
+		Message:    "Check ScorecardBasicSpecCheck encountered an error. Please review the " + scorecardBasicCheckResult + " file in your execution artifacts for more information.",
```

Comment:
These messages referenced a filesystem path in the past. I'm not sure this makes sense anymore.

Reply:
Is `artificats.Path()` a thing still? I'm guessing the answer is in some cases yes, in some cases no....so it's probably safe to remove this to avoid `nil` pointer panics, even if the log is less clear now for some use cases.

Reply:
Technically No.

`artifacts.Path()` was a function in the Artifacts package that would return the singleton's configured directory (previously set by artifacts.SetDir()`. 

The FilesystemWriter does have a `Path()` method, and that's what we use in CLI invocations. This result, however, is returned to the user regardless of their context (e.g. library calls, cli exec, whatever).

Checks have access to an ArtifactWriter interface via the context, and that interface only defines the `WriteFile` method. I initially had the `Path` method defined to solve this use case, but other artifact writers don't necessarily have this concept of a path, so I did away with it here to make sure that our results are caller agnostic.

Hope that makes sense.

Reply:
That makes perfect sense.